docs/source/_templates/layout.html

```diff
@@ -1,5 +1,26 @@
 {% extends "pydata_sphinx_theme/layout.html" %}
 
+{# Use Matomo #}
+{% block extrahead %}
+<!-- Matomo -->
```

Comment:
This might need a `{{ super() }}` to keep not replace the full content of the `extrahead` block of the parent theme, but to only append the matomo script?

(the extrahead block still seems to have some useful things (eg favicons) apart from the google analytics)

Reply:
Thanks! I didn't know it. I've added.